Themes/default/css/custom.css

```diff
@@ -0,0 +1,787 @@
+/**
```

Comment:
Do we have to name this custom.css?  I've typically used that for when I've customized a theme.  Makes it seem odd that I have a custom.css with changes that are default in the theme.

Reply:
In fact it should be icons.css

Reply:
Also the header can be removed...
I can look into these things later this month though, I suppose I can PR to live repo

Reply:
> In fact it should be icons.css

Agreed

Reply:
> I just now noticed that this file is also appended to index.css, strangely enough.

Huh... can't remember why I did that.  I was probably thinking about just removing the extra file... and never did.